packages/mds-transactions-api/api/index.ts

```diff
@@ -28,17 +34,32 @@ export const api = (app: express.Express): express.Express =>
       GetTransactionHandler
     )
     .post(
-      pathPrefix('/transactions'),
+      pathPrefix('/transaction'),
       checkTransactionApiAccess(scopes => scopes.includes('transactions:write')),
       CreateTransactionHandler
     )
-    .put(
-      pathPrefix('/transactions/:transaction_id'),
+    .post(
+      pathPrefix('/transactions'),
       checkTransactionApiAccess(scopes => scopes.includes('transactions:write')),
-      UpdateTransactionHandler
+      CreateTransactionsHandler
     )
-    .delete(
-      pathPrefix('/transactions/:transaction_id'),
+    .get(
+      pathPrefix('/transactions/operations'),
```

Comment:
You probably want `:transaction_id` in the route here now, right? `/transactions/:transaction_id/operations`